content/docs/sidebar.json

```diff
@@ -101,8 +100,9 @@
         "source": false,
         "children": [
           "stop-tracking-data",
-          "update-tracked-files",
-          "add-deps-or-outs-to-a-stage"
+          "update-tracked-data",
+          "add-deps-or-outs-to-a-stage",
+          "merge-conflicts"
         ]
       },
       "setup-google-drive-remote",
```

Comment:
Feels like this lonely guide could also be a how-to, at least until we expand on all the other types and figure out where to put them.

Reply:
> You can put pretty much anything into How to this way ... How to manage external data, How to contribute, How to optimize deal with large data, How to run on Windows, etc ...
>
> I think there should be something else that defines this section - closer to FAQ? some specific workflow questions?
>
> In case of Google Drive - we'll have one page per remote. Even size in this case makes a bad fit. Eventually it might become bigger than everything else.

putting "how to" before the title doesn't make it a how=to, the content has to be short recipes to address specific problems. The ones you mention are 50% + explanation, the rest examples so that doesn't really count (:

How to setup remotes also includes some explanation, but mostly how-to stuff.

Reply:
> How to Setup Remotes also includes some explanation, but mostly how-to stuff.

I guess you could argue it's a lot of explanation actually. Ok, let's leave it as a guide!